.github/instructions/mastg-apps.instructions.md

```diff
@@ -10,9 +10,9 @@ Standards for authoring reference application pages under `apps/`. These pages d
 
 ## File naming and IDs
 
-- The app ID is defined by the filename: `MASTG-APP-\d{4}.md`
+- The filename defines the app ID: `MASTG-APP-\d{4}.md`
 - Do not add an `id:` field to the YAML front matter
-- Use the next available number within the platform folder. Coordinate in PRs to avoid collisions
+- Use the following available number within the platform folder. Coordinate in PRs to avoid collisions
```

Comment:
Somehow `next` feels more accurate. 

```suggestion
- Use the next available number in the platform folder. Coordinate in PRs to avoid collisions
```